tests/test_migrations.py

```diff
@@ -0,0 +1,26 @@
+import pytest, os
+import inspect as ins
+from sqlalchemy import inspect, create_engine 
+from alembic.config import Config
+from alembic import command
+from flask import current_app
+from models import User
+
+
+this_file_directory = os.path.dirname(os.path.abspath(ins.stack()[0][1]))
+root_directory      = os.path.join(this_file_directory, '..')
+alembic_directory   = os.path.join(root_directory, 'alembic')
+ini_path            = os.path.join(root_directory, 'alembic.ini')
+
+alembic_cfg = Config(ini_path)
+alembic_cfg.set_main_option('script_location', alembic_directory) 
+
+def test_alembic_downgrade_upgrade_migrations(app):
+    command.downgrade(alembic_cfg, 'base')
+    with app.app_context():
+        insp_1 = inspect(current_app.engine)
+        assert 'users' not in insp_1.get_table_names()
+        command.upgrade(alembic_cfg, 'head')
+        insp_2 = inspect(current_app.engine)
+        assert 'users' in insp_2.get_table_names()
+        assert User().__table__.columns.keys() == [insp_2.get_columns('users')[i]['name'] for i in range(len(insp_2.get_columns('users')))]
```

Comment:
This can be simplified to
```
assert User().__table__.columns.keys() == [column['name'] for column in insp_2.get_columns('users')]
```
and it also makes sense to move it into a separate function with a descriptive name.

Reply:
Done!